common/source/docs/common-future-wiki-changes.rst

```diff
@@ -352,6 +352,17 @@ Manual Forward Throttle in VTOL Modes
 
 By setting an RC channel option (``RCx_OPTION``) to "209", that channel can provide a separate throttle input to the forward motor(s) in QSTABILIZE, QACRO, and QHOVER VTOL modes. This allows forward movement without having to tilt the QuadPlane forward requiring throttle stick repositioning in QSTABILIZE and QACRO to maintain altitude, and present more forward flat plate resistance to forward movement in all modes.
 
+On QuadPlane Parameters page:
+-----------------------------
+
+-add to Q_OPTIONS
+-  bit 6, if set, will enforce the ICE idle governor even in MANUAL mode.
+-  bit 7, if set, will force QASSIST to be active at all times in VTOL modes. See :ref:`Assisted Fixed-Winf Flight<assisted_fixed_wing_flight>`.
+-  bit 8, if set, will prevent control surface movements in Tailsitters when QASSIST (:ref:`Assisted Fixed-Winf Flight<assisted_fixed_wing_flight>` ) is active.
```

Comment:
QASSIST takes over the motors only. The control surfaces are left on the plane controller. QASSIST without this the copter controller uses both the  motors and control surfaces, the plane controller does nothing. With this the copter does motors and plane does surfaces.

Reply:
yep, poor choice of words...perhaps its clearer now